src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ToolStripOverflow.ToolStripOverflowAccessibleObjectTests.cs

```diff
@@ -79,5 +79,147 @@ public void ToolStripOverflowAccessibleObject_GetPropertyValue_ControlType_IsExp
             Assert.Equal(expected, actual);
             Assert.False(toolStripOverflow.IsHandleCreated);
         }
+
+        [WinFormsTheory]
+        [InlineData((int)UiaCore.NavigateDirection.NextSibling)]
+        [InlineData((int)UiaCore.NavigateDirection.PreviousSibling)]
+        [InlineData((int)UiaCore.NavigateDirection.FirstChild)]
+        [InlineData((int)UiaCore.NavigateDirection.LastChild)]
+        public void ToolStripOverflowAccessibleObject_FragmentNavigate_ReturnsNull_IfHandleNotCreated(int navigateDirection)
+        {
+            using ToolStripButton toolStripItem = new ToolStripButton();
```

Comment:
```suggestion
            using ToolStripButton toolStripItem = new();
```
Please enforce this pattern in this file